src/platforms/x86_64/architecture.c

```diff
@@ -0,0 +1,34 @@
+#include "platforms.h"
+
+Platform platform_x86_64_sysv = {.name = "x86_64-sysv",
+                                 .word_size = 8,
+
+                                 .num_register_classes = 2,
+                                 .register_classes = (RegisterClass[]){
+                                     {.reg_class = PLATFORM_REG_GENERAL_PURPOSE,
+                                      .num_registers = 16,
+                                      .registers =
+                                          (PlatformRegister[]){
+                                              {.name = "ax", .num = 0},
+                                              {.name = "cx", .num = 1},
+                                              {.name = "dx", .num = 2},
+                                              {.name = "si", .num = 3},
+                                              {.name = "di", .num = 4},
+                                              {.name = "bx", .num = 5},
```

Comment:
These should probably ``rax``, ``rcx``, ``rdx``, etc because you are using the full 64 bit names for ``r11`` and friends.